misc/dist/ios_xcode/godot_ios.xcodeproj/project.pbxproj

```diff
@@ -100,7 +100,7 @@
 		D0BCFE2B18AEBDA2004A7AAE = {
 			isa = PBXGroup;
 			children = (
-				87D30C9722235A2200E688DC /* Config.xcconfig */
+				87D30C9722235A2200E688DC /* Config.xcconfig */,
```

Comment:
Would be nice if you could squash this commit into its parent (40a2122b).

Reply:
and/or group some of the other related commits that iteratively modify the same file.